src/llmcompressor/modifiers/quantization/calibration.py

```
@@ -86,20 +86,34 @@ def initialize_observer(
     :param base_name: substring used to fetch the observer and value to observe
+    :param observe_all_fused: whether to also observe fused obs
     """
+    obs_name = f"{base_name}_observer"
+    
+    if observe_all_fused:
+        modules = set(module) if isinstance(module, Iterable) else {module}
+        all_modules = {} | modules
+        for mod in modules:
+            if not hasattr(mod, obs_name):
+                continue
+            all_modules |= set(getattr(mods, obs_name)._fused_observers)
```

Ruff (0.15.13)

[error] Undefined name `mods`

(F821)